tests/commoncode/test_fileutils.py

```diff
@@ -318,14 +318,14 @@ def test_os_walk_with_unicode_path(self):
     def test_fileutils_walk(self):
         test_dir = self.get_test_loc('fileutils/walk')
         base = self.get_test_loc('fileutils')
-        result = [(as_posixpath(t.replace(base, '')), d, f,) for t, d, f in fileutils.walk(test_dir)]
+        result = [(as_posixpath(t.replace(base, '')), d, sorted(f),) for t, d, f in fileutils.walk(test_dir)]
```

Comment:
ok, this make sense. Can you put that in a completely different pull request?
I would rather not mix this with the package model. They are completely unrelated things.